src/about/index.html

```diff
@@ -13,7 +13,7 @@ <h1 class="intro-section__headline">About Us</h1>
   <div class="section-content">
     <h1 class="about-page__heading">About DecodeMTL</h1>
     <p class="about-page__text">
-      We are currently running our courses out of the GriffinCamp a leading startup accelerator in downtown Montreal's innovation district.
+      We are currently running our courses out of "La Commune" a leading startup accelerator in downtown Montreal's innovation district.
       Our goal is to bring coding literacy to as many people as possible. We believe that with the right environment, best teachers,
```

Comment:
Can you find a better description of La Commune please? Changing the name but keeping the old description seems wrong.